docs/administration/configuration/configuring-external-access/index.md

```diff
@@ -0,0 +1,136 @@
+# Configuring external access
+
+REANA manages external access with the use of Kubernetes [Ingresses](https://kubernetes.io/docs/concepts/services-networking/ingress/).
```

Comment:
... manages external traffic to the cluster with the use of...

Perhaps?  (The access nomenclature also depends on how we solve the page naming issue mentioned above.)

Reply:
Yes, I also prefer _external traffic_ to _external access_